### PR TITLE
Add support to trigger OCP installs from UMB message

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Adds a scale-ci build tracker job which queries the UMB message
  to capture the build, payload info and copies it to the specified
  destination.
- Adds support to the OCP AWS and Azure jobs to use the info from the
  scale-ci build tracker when the trigger option is set.

### CODE REVIEW

File: pipeline-scripts/scale_ci_build_tracker.groovy [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+						[$class: 'StringParameterValue', name: 'ORCHESTRATION_USER', value: orchestration_user ],
+						[$class: 'StringParameterValue', name: 'SSHKEY_TOKEN', value: sshkey_token ],
+						[$class: 'StringParameterValue', name: 'BUILD_INFO_URL', value: build_info_url ],
+						[$class: 'StringParameterValue', name: 'BUILD_INFO_DESTINATION', value: build_info_destination ]]  
+			} catch ( Exception e) {
+				echo "SCALE-CI-BUILD-TRACKER Job failed with the following error: "
+				echo "${e.getMessage()}"
+				echo "Sending an email"
+				mail(
+					to: 'nelluri@redhat.com',
```
Comment (jtaleric) [inline pipeline-scripts/scale_ci_build_tracker.groovy]: can we create a mailer? `perfscale-ci-mailer`?

**Comment by chaitanyaenr** [comment]
@jtaleric Sure, will add the list of emails. Thanks.

**Comment by chaitanyaenr** [comment]
This PR depends on https://github.com/openshift-scale/scale-ci-deploy/pull/58. The install jobs need to be updated to look at the status file with PROCEED/STOP signal to decide whether to cleanup and reinstall using the captured payload.

File: jjb/dynamic/scale-ci-pipeline.yml [hunk 2]
```diff
@@ -9,6 +9,10 @@
         default: true
         description: ''
         name: SCALE_CI_WATCHER
+    - bool:
+        default: true
```
Comment (jtaleric) [inline jjb/dynamic/scale-ci-pipeline.yml]: Should this default to false?

**Comment by chaitanyaenr** [comment]
@jtaleric The current PR supports both workflow 1 and 2 provided the trigger vars are defined but not set. To build a job using the workflow 1, we need to set SCALE_CI_BUILD_TRIGGER=false
SCALE_CI_BUILD_TRIGGER_URL= in the properties file. This will just skip the trigger check and will use the build specified in the properties file.

**Comment by jtaleric** [comment]
> @jtaleric The current PR supports both workflow 1 and 2 provided the trigger vars are defined but not set. To build a job using the workflow 1, we need to set SCALE_CI_BUILD_TRIGGER=false
> SCALE_CI_BUILD_TRIGGER_URL= in the properties file. This will just skip the trigger check and will use the build specified in the properties file. There were some issues with the job not interpreting the vars correctly, the latest commit addresses it by explicitly type casting them.

So, in `workflow 1`, if anyone has their properties file and attempt to run after this change, it will fail. Since their properties file doesn't have the two vars. 

`workflow 1` shouldn't depend on anything for `workflow 2`, they should be independent. IMHO.

**Pull request merged**